### PR TITLE
Alerts

### PR DESCRIPTION
**Pull request by ethanjurman**
Icons on drawer
image

Alert Details and Location Pages
image

### CODE REVIEW

**Comment by ethanjurman** [comment]
:facepunch: TESTS PASS. GET RECK'D :facepunch:

File: www/js/components/add-alert-details.js [hunk 1]
```diff
@@ -0,0 +1,94 @@
...
+    setPointName( name );
+  }
+
+  onDescriptionUpdate( event ) {
+    const {setPointDescription} = this.props;
+    const description = event.target.value;
+    setPointDescription( description );
+  }
+
+  onPhotoAdd() {
```
Comment (JRJurman) [inline www/js/components/add-alert-details.js]: did we actually want to do photos for alerts?

File: www/js/components/block.js [hunk 2]
```diff
@@ -80,16 +80,16 @@ export class FormBlock extends Component {
...
+                    <div className='entry__spacer' />
+                    <RaisedButton secondary
+                      className='entry__action'
+                      fullWidth={ true }
+                      onClick={ this.onAction }
+                      label={ this.props.actionText } />
       </Block>
       );
-    /*esfmt-ignore-end*/
+      /*esfmt-ignore-end*/
```
Comment (JRJurman) [inline www/js/components/block.js]: wat?

**Comment by JRJurman** [comment]
Do the changes here remove alerts as a field in redux, and are handled in filter?

File: www/js/components/add-alert-details.js (hunk 1 above)
Comment (ethanjurman) [inline www/js/components/add-alert-details.js]: We'll discuss this thursday, but we should highlight this as being optional (as with other fields)

File: www/js/components/block.js (hunk 2 above)
Comment (ethanjurman) [inline www/js/components/block.js]: I'll fix this soon :+1:

**Comment by JRJurman** [comment]
:tada:

**Pull request merged**